lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -77,24 +78,29 @@ def main():
     # declare the pedestal calibrator
     lst_r0 = LSTR0Corrections(pedestal_path=args.pedestal_file, config=config)
 
+    reader = event_source(input_url=path_list[0], max_events=args.max_events)
```

Comment:
shouldn't this `event_source` be substituted by `LSTEventSource`?
At least should be considered for the future:
https://github.com/cta-observatory/ctapipe/issues/1177#issuecomment-555429621

Reply:
Yes. I changed to `LSTEventSource`